spyder/plugins/editor/widgets/editorstack/tests/test_save.py

```diff
@@ -515,14 +515,16 @@ def test_save_when_completions_are_visible(completions_editor, qtbot):
     assert "some" in [x['label'] for x in sig.args[0]]
     assert "something" in [x['label'] for x in sig.args[0]]
 
-    # Press keyboard shortcut corresponding to save
-    qtbot.keyPress(
-        completion, Qt.Key_S, modifier=Qt.ControlModifier, delay=300)
-
-    # Assert file was saved
-    with open(file_path, 'r') as f:
-        saved_text = f.read()
-    assert saved_text == 'some = 0\nsomething = 1\nsome'
+    # Check that pressing Ctrl-S emits the signal for saving the file
```

Comment:
```suggestion
    # Check that pressing Ctrl+S emits the signal for saving the file
```

To follow the docstring text.